Mathlib/RepresentationTheory/Tannaka.lean

```diff
@@ -147,6 +147,56 @@ def algHomOfRightFDRepComp (η : Aut (forget k G)) : (G → k) →ₐ[k] (G →
   apply_fun (fun x ↦ (x.hom.app rightFDRep).hom (1 : G → k)) at this
   exact this
 
+variable [DecidableEq G]
+
+/-- Auxiliary map for the proof of `toRightFDRepComp_inj`. -/
+@[simps]
+def auxLinearMap {X : FDRep k G} (v : X) : (G → k) →ₗ[k] X where
```

Comment:
```suggestion
def _root_.FDRep.sumSMulInv {X : FDRep k G} (v : X) : (G → k) →ₗ[k] X where
```
I'm definitely not a fan of `auxLinearMap`, I'm not even sure this declaration should be in the TannakaDuality namespace at all.